USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -182,6 +181,29 @@ private void RenderMarker(Marker input, StyleConfig styles, XWPFParagraph parent
                         RenderMarker(marker, markerStyle, parentParagraph);
                     }
                     break;
+
+            // Table Markers
+                case TableBlock table:
+                    XWPFTable tableContainer = newDoc.CreateTable();
+
+                    // Clear Borders
+                    tableContainer.SetBottomBorder(XWPFTable.XWPFBorderType.NONE,0,0,"#FFFFFFF");
```

Comment:
is the color of the border needed if you're hiding it?

Reply:
Tables are preset with black borders and I didn't find a method to set no Borders.

Reply:
I mean you're setting `XWPFTable.XWPFBorderType.NONE` so I would guess that sets it to none?

Reply:
@rbnswartz there's no overloads for this function...I have to fill out each param. There's also no setter for BorderType.

Reply:
Ok you needed to add the color. Ok that makes sense